Makefile

```diff
@@ -36,8 +36,20 @@ apply-with-kind: verify build-with-kind ## Deploy the kwok controller from the c
 		$(HELM_OPTS) \
 		--set controller.image.repository=$(IMG_REPOSITORY) \
 		--set controller.image.tag=$(IMG_TAG) \
+		--set serviceMonitor.enabled=true \
 		--set-string controller.env[0].name=ENABLE_PROFILING \
-		--set-string controller.env[0].value=true
+		--set-string controller.env[0].value=true 
+
+e2etests: ## Run the e2e suite against your local cluster
+	cd test && go test \
+		-count 1 \
+		-timeout 30m \
```

Comment:
I can't remember, but I feel like I might have validated that setting just one of `timeout` and `ginkgo.timeout` was sufficient to get the timeout to work

Reply:
I'm hoping to just keep it as is to mimic our e2e tests in aws provider